pkg/webhooks/mutator/mutator_test.go

```diff
@@ -53,15 +53,15 @@ var _ = Describe("webhooks mutator", func() {
 		sspv1beta1.AddToScheme,
 		corev1.AddToScheme,
 	} {
-		Expect(f(s)).To(BeNil())
+		Expect(f(s)).To(Succeed())
 	}
 
 	codecFactory := serializer.NewCodecFactory(s)
 	corev1Codec := codecFactory.LegacyCodec(corev1.SchemeGroupVersion)
 
 	Context("Check mutating webhook for namespace deletion", func() {
 		BeforeEach(func() {
-			Expect(os.Setenv("OPERATOR_NAMESPACE", HcoValidNamespace)).To(BeNil())
+			Expect(os.Setenv("OPERATOR_NAMESPACE", HcoValidNamespace)).To(Succeed())
```

Comment:
Are you sure this is the most appropriate matcher in this case?
From what I understood, `Succeed()` matcher should be used with a function that returns an error-type as its first or only return value.
https://onsi.github.io/gomega/#succeed

Reply:
Totally agree. This is what the linter looks for, and here is the os.Setenv code from the standard library:

```golang
func Setenv(key, value string) error {
	err := syscall.Setenv(key, value)
	if err != nil {
		return NewSyscallError("setenv", err)
	}
	return nil
}
```

Reply:
ok, cool 👍